tests/framework/TestXSD/TestFails.py

```diff
@@ -25,17 +25,19 @@
 import sys, os
 from lxml import etree
 
-ravenDir = os.path.dirname(os.path.dirname(os.getcwd()))
+testDir = os.getcwd()
+print("testDir", testDir)
```

Comment:
This is left here on purpose, right?

Reply:
Yes, it at least helps debugging path problems.  (Even tho' the failure was ultimately because of a missing prereq in the tests file)